bvm/ballerina-profiler/src/main/java/io/ballerina/runtime/profiler/Profiler.java

```diff
@@ -159,6 +161,14 @@ private void handleProfilerArguments(String[] args) {
                     this.targetDir = args[i + 1];
                     addToUsedArgs(args, usedArgs, i);
                 }
+                case "--sourceroot" -> {
+                    this.sourceRoot = args[i + 1];
+                    addToUsedArgs(args, usedArgs, i);
+                }
+                case "--pDebug" -> {
```

Comment:
```suggestion
                case "--debugProfiler" -> {
```

Reply:
Can't we use --debug for profiler runtime too?

Reply:
This flag is for internal use. We pass them internally when the user sets the env variable `BAL_PROFILE_JAVA_DEBUG` for debugging the `Profiler.jar`. And it can clash with the debugging of `temp.jar` if we use `--debug`.